server/channels/app/platform/web_hub.go

```diff
@@ -92,8 +92,7 @@ func newWebHub(ps *PlatformService) *Hub {
 
 // hubStart starts all the hubs.
 func (ps *PlatformService) hubStart(broadcastHooks map[string]BroadcastHook) {
-	// Total number of hubs is twice the number of CPUs.
-	numberOfHubs := runtime.NumCPU() * 2
+	numberOfHubs := runtime.NumCPU()
```

Comment:
Nit: maybe we should add a comment linking to this PR to explain why this number was chosen.

Reply:
+1, let's keep the rationale documented this time, link to test results and discussion would be good.